app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 # Represents a user in the application. Users are shared across all instances.
 class User < ActiveRecord::Base
   include UserAuthenticationConcern
+  include UserOmniauthConcern
```

Comment:
You can fix this by including the UserOmniauthConcern at the end of the UserAuthenticationConcern included block. This is because including in the module includes the module before devise. so devise is called and not you.

anyway there should be a model spec that checks for this.